database/changesets/20210422_TerraWorkspaceSinkTable.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="chengche@broadinstitute.org"
+               dbms="postgresql"
+               id="20210422"
+               logicalFilePath="20210422_TerraWorkspaceSinkTable.xml"
+               runInTransaction="false">
+        <comment>
+            the 'Terra Workspace' sink table schema
+        </comment>
+        <sql>
+            CREATE TYPE TerraWorkspace AS (
+                id              bigint,      -- unique to this table
+                workspace       text,        -- the fully-qualified 'Namespace/Name' of the Terra workspace to execute workflows in
+                entity          text,        -- the entity type to import data from the source as
+                fromOutputs     text         -- mapping of how to coerce the output of executor to sink
```

Comment:
I'm not sure about this

Reply:
probably fine. if it isn't we can change it